Update mdpi template

nielsbock opened the pull request:
Updated files for the MDPI template to the 21 March 2024 version (https://www.mdpi.com/authors/latex). Minor edits to accommodate the new files.

cderv commented:
Thanks ! 

We have CI error related to mdpi in main, so we should try to fix in this PR 
- https://github.com/rstudio/rticles/issues/574

nielsbock commented:
I think the error came from the springer template. I added `\orcidlink` in my last PR. It wasn't a part of the official template but something i thought was nice to have. However, `orcidlink` has recently been updated (https://github.com/duetosymmetry/orcidlink-LaTeX-command/commit/fe2e6eef0fe07a9ba0334d42aaa72419e4c84cb6) which deprecates the code causing this error, so I just removed it again.

nielsbock commented:
I don't know what the remaining check fail is about.
